libs/dh/globalization/assets-localization/src/assets/i18n/da.json

```diff
@@ -1,3 +1,14 @@
 {
-  "appName": "DataHub [DA]"
+  "appName": "DataHub [DA]",
+  "metering-point": {
+    "search": {
+      "title": "Målepunkter",
+      "search-label": "Indtast et Målepunkts ID for at fremsøge et målepunkt",
+      "search-placeholder": "Søg på Målepunkts ID",
+      "search-button": "Søg",
+      "search-invalid-length": "Målepunkts ID’et skal være 18 cifre",
+      "no-metering-point-found-title": "Vi fandt Desværre intet målepunkt",
+      "no-metering-point-found-message": "Tjek for eventuelle tastefejl og prøv igen"
+    }
+  }
```

Comment:
question: should we use *camelCase* as suggested by Transloco and used by `appTitle` or *kebab-case* as here?

Reply:
If the suggested way of doing it is camelCase, I think we should do that.